### DISCUSSION TITLE
Need Help on converting python script  to exe- (script that uses Custom_Widgets module)

### DISCUSSION THREAD

**Discussion by ovclara** [KhamisiKibet/QT-PyQt-PySide-Custom-Widgets#54] [Q&A]
Hello,

I have an GUI application build with help of Custom_Widgets module. 
Thanks to Custom_Widgets module, it is really cool and easy to build quality and modern UI .
But, when I try to build a executable out of it with pyinstaller or cx_freeze, the exe do not work.
It some how do not find the Custom_Widgets module.

Can any one help on this ? has this been tried by someone?

**Comment by KhamisiKibet** [comment]
Which error are you getting?

**Reply by pdnguyen179** [reply]
i got this: File "Custom_Widgets\Widgets.py", line 3097, in updateAppSettingsAttributeError: 'MainWindow' object has no attribute 'orginazationName'

**Reply by ovclara** [reply]
Once I build the exe , while launching exe-  I get error- "Module Custom_Widgets not found'.
I tried to add it as hidden-import as well but did not worked.

Finally, I was not able to used Custom_Widgets module - I have to manually manage the animations and stuff.

If you have any solution for bundling this module into exe, Please share.

**Comment by Danito90** [comment]
I have the same problem, when trying to create an .exe of an application in which I use the QT-PyQt-PySide-Custom-Widgets library, all the windows work but not the animations that are made by the style.json... like It can be fixed? How do you create your .exe with this library?
